.github/ISSUE_TEMPLATE/new-ontology-request.md

```diff
@@ -49,3 +49,28 @@ GitHub username:
 
 
 ## Additional comments or remarks
+
+## Metadata
+
+Please fill in the following metadata record which will be used by the OBO Foundry website.
+
+```
+id: yourfourletterid
+title: The Title Of Your Ontology
+contact:
+  email: email@email.fr
+  label: Your Name
+description: Some one sentence description of your ontology.
+domain: your_domain_like_for_example_anatomy
+homepage: https://github.com/EBISPOT/amphx_ontology
+products:
+  - id: yourfourletterid.owl
+  - id: yourfourletterid.obo
+dependencies:
+  - id: ro
+  - id: otheroboontology
+tracker: https://github.com/YOURORG/your_repo/issues
+license:
+  url: http://creativecommons.org/licenses/by/3.0/
```

Comment:
is there a way to (succinctly) make it clearer here that this is just an example, and not the only license we accept?

Reply:
Not succinctly, but I added some comments up top. Can you check again?